src/main/scala/net/ruippeixotog/scalascraper/util/ProxyUtils.scala

```diff
@@ -0,0 +1,32 @@
+package net.ruippeixotog.scalascraper.util
+
+object ProxyUtils {
+
+  /**
+   * Set HTTP and HTTPS proxy JVM-wide
+   */
+  def setProxy(host: String, port: Int): Unit = setProxy(Some(host), Some(port))
+
+  /**
+   * Remove HTTP and HTTPS proxy configuration JVM-wide
+   */
+  def removeProxy(): Unit = setProxy(None, None)
+
+  /**
+   * Implicit conversions to String or Java null
+   */
+  private implicit def optionStrToJavaStr(value: Option[String]): String = value.getOrElse(null)
```

Comment:
Yes you are right here! In fact the code for removing the proxy it wasn't working. I should use System.clearProperty(key). Check the last commit!